guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -137,5 +137,44 @@ include::snip_steps-needs-reboot.adoc[]
 
 . Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups made earlier.
 ifdef::orcharhino,satellite[]
-. Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
+. Optional: If you use custom repositories,ensure that you enable these custom repositories after the upgrade completes.
 endif::[]
+
+.Upgrading {SmartProxyServersTitle} Using Remote Execution
+
+. Create a backup.
++
+* On a virtual machine, take a snapshot.
+* On a physical machine, create a backup.
++
+For more information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
+
+. Optional: If you made manual edits to DNS or DHCP configuration in the `/etc/zones.conf` or `/etc/dhcp/dhcpd.conf` files, back up the configuration files.
+The installer only supports one domain or subnet, and therefore restoring changes from these backups might be required.
+
+. Optional: If you made manual edits to DNS or DHCP configuration files and do not want to overwrite the changes, enter the following command:
++
+[options="nowrap"]
+----
+# {foreman-installer} --foreman-proxy-dns-managed=false \
+--foreman-proxy-dhcp-managed=false
+----
+
+. In the {ProjectWebUI} , navigate to *Monitor* > *Jobs*.
+
+. Click *Run Job*.
+
+. From the *Job category* list, select *Maintenance Operations*.
+
+. From the *Job template* list, select *{SmartProxy} Upgrade Playbook*.
+
+. In the *Search Query* field, enter the host name of the {SmartProxy}.
+
+. Ensure that *Resolves to* shows *1 host*.
+
+. In the *target_version* field, enter the target version of the {SmartProxy}.
+
+. In the *whitelist_options* field, enter the whitelist options.
```

Comment:
Now that's interesting. I am doing an inclusive language audit of the docs (SAT-23271) and did not think to check for 'whitelist' because it is so seldom used. Obviously, we can't change the field name. I recommend you change "enter the whitelist options" to "enter your options" or similar. 

Another possibility is not to mention the fields at all. It is acceptable to write "Enter the target version of the {SmartProxy}" or "Click **Static Query**" or "Select the schedule..." if the UI elements are clear.

Reply:
Agreed. Changed to `enter your options` for now,